tests/integration-tests/tests/test_efa/test_efa.py

```diff
@@ -71,12 +72,14 @@ def _test_efa_mpi(remote_command_executor, scheduler_commands, test_datadir, slo
     )
 
     # submit script using additional files
-    result = scheduler_commands.submit_script(str(test_datadir / "mpi_submit.sh"), slots=2 * slots_per_instance)
+    result = scheduler_commands.submit_script(
+        str(test_datadir / "mpi_submit.sh"), slots=2, filename="mpi_submit.sh.o%j"
```

Comment:
filename was defined in submit_command for slurm and submit_script for sge 

Also Filename is not used in sge implementation. If it’s easier to have cross schedulers compatibility you can leave the redirection on the mpirun command.

Reply:
I've changed it to use `>>`, problem before was one mpi process output nothing, which cause the file to be empty. New behavior is each mpi process appends to the same file.